packages/salesforcedx-vscode-apex-debugger/src/adapter/debugConfigurationProvider.ts

```diff
@@ -97,9 +97,15 @@ export class DebugConfigurationProvider
     while (
       this.sfdxApex &&
       this.sfdxApex.exports &&
-      !this.sfdxApex.exports.isLanguageClientReady() &&
+      !this.sfdxApex.exports.languageClientStatus.isReady() &&
       !expired
     ) {
+      if (this.sfdxApex.exports.languageClientStatus.failedToInitialize()) {
```

Comment:
This new check allows us to surface a better error message when the user triggers the debugger. We used to display a language server is not ready message. We will now display the correct error that caused the Apex language server to be unavailable.

![debugger_error](https://user-images.githubusercontent.com/1041842/56084045-531e5680-5de2-11e9-89ca-080e535fad93.png)

Reply:
If this can't be a link, I'd remove the square brackets, the parentheses, and the last period:
`see Set Your Java Version: https://...`

Reply:
This is the same error shown in the info message. These changes now keep it around in order to show it when a user tries to execute the debuggers or anything else depending on the Apex language server. If I modify it then it means the info messages won't open a browser window when the user clicks on the link.

Reply:
Ok, that makes sense.